util/inc/leatherman/util/scoped_descriptor.hpp

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
@Magisus I think I agree that this is clearer in util/posix. We may want to re-arrange posix-specific and windows-specific utilities sometime to be more consistent (we have a windows module and then a posix-specific class in the util module).